### ISSUE TITLE
State Wrapper fuzz errors

### ISSUE THREAD

**Issue by marlitas** [phetsims/center-and-variability#426]
While working on https://github.com/phetsims/center-and-variability/issues/421, I hit multiple other errors while fuzzing in the state wrapper. Here is a list of some that popped up for me but there may be more. We probably want to do a State wrapper sprint to figure out these bugs. I'm not the best at tracking these down so some help would be really appreciated.

Errors:

- `String should be defined and non-null. Use the empty string if needed.`

| window.assertions.assertFunction  | assert.js:24 |
| -- | -- |
| setString | Text.ts:143 |
| set string | Text.ts:153 |
| updateIQRNode | IQRNode.ts:320 |
| emit | TinyEmitter.ts:122 |
| emit | Emitter.ts:61 |
| updateDataMeasures | VariabilitySceneModel.ts:157 |
| updateDataMeasures | SoccerSceneModel.ts:135 |
| (anonymous) | SoccerSceneModel.ts:288 |

- `Assertion failed:  The intervalToolValue2Property is outside of it's range: 17.058186475409837`

(anonymous) | @ | IntervalToolNode.ts:132
-- | -- | --
| emit | @ | TinyEmitter.ts:122
| _notifyListeners | @ | ReadOnlyProperty.ts:315
| unguardedSet | @ | ReadOnlyProperty.ts:262
| set | @ | ReadOnlyProperty.ts:246
| onPropertyPropertyChange | @ | DynamicProperty.ts:232
| emit | @ | TinyEmitter.ts:122
| _notifyListeners | @ | ReadOnlyProperty.ts:315
| (anonymous) | @ | ReadOnlyProperty.ts:363
| attemptToApplyPhases | @ | PropertyStateHandler.ts:281
| undeferAndNotifyProperties | @ | PropertyStateHandler.ts:201
| (anonymous) | @ | PropertyStateHandler.ts:96

**Comment by samreid** [comment]
Also:

```
center-and-variability : phet-io-state-fuzz : unbuilt
http://127.0.0.1/continuous-testing/ct-snapshots/1691522169999/phet-io-wrappers/state/?sim=center-and-variability&phetioDebug=true&phetioWrapperDebug=true&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22center-and-variability%22%2C%22phet-io-state-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1691522169999%22%2C%22timestamp%22%3A1691523005738%7D
Uncaught Error: Assertion failed: A rectangle needs to have a non-negative finite width (-108.92857142857144)
Error: Assertion failed: A rectangle needs to have a non-negative finite width (-108.92857142857144)
at window.assertions.assertFunction (http://127.0.0.1/continuous-testing/ct-snapshots/1691522169999/assert/js/assert.js:28:13)
at assert (Rectangle.ts:626:14)
at invalidateRectangle (Rectangle.ts:391:11)
at setRectWidth (Rectangle.ts:396:47)
at (IQRNode.ts:263:29)
at listener (TinyEmitter.ts:122:8)
at emit (Emitter.ts:61:21)
at emit (VariabilitySceneModel.ts:158:49)
at updateDataMeasures (SoccerSceneModel.ts:132:42)
at updateDataMeasures (SoccerSceneModel.ts:271:10)
id: Sparky Puppeteer
Snapshot from 8/8/2023, 1:16:09 PM
```

**Comment by samreid** [comment]
I'm also hitting a soccer ball that is classified as landed and stacked, but it has a null value (state wrapper):

```
Client.ts:2047 Uncaught (in promise) Error: Assertion failed: String should be defined and non-null. Use the empty string if needed.
    at window.assertions.assertFunction (assert.js:28:13)
    at Text.setString (Text.ts:143:15)
    at set string [as string] (Text.ts:153:54)
    at MeanAndMedianInfoNode.ts:50:35
    at TinyEmitter.emit (TinyEmitter.ts:122:9)
    at Emitter.emit (Emitter.ts:61:22)
    at SoccerSceneModel.ts:536:35
    at TinyEmitter.emit (TinyEmitter.ts:122:9)
    at Emitter.emit (Emitter.ts:61:22)
    at Animation.step (Animation.ts:322:25)
```

**Comment by samreid** [comment]
We added earlier assertions. We fixed problems in #202. We will open new issues if CT shows us there is more to do. Closing.
